README.md

```diff
@@ -37,7 +37,7 @@ Using [npm](https://www.npmjs.com/):
 
 Then with a module bundler like [webpack](https://webpack.github.io/) that supports either CommonJS or ES2015 modules, use as you would anything else:
 
-```js
+```jsx
```

Comment:
This example isn't using jsx at all. Not sure you're getting much value out of putting `jsx` here :)